test/integration/controlplane/transformation/kms_transformation_test.go

```diff
@@ -279,6 +281,470 @@ resources:
 	}
 }
 
+// TestECHotReload is an integration test that verifies hot reload of KMS encryption config works.
+// This test asserts following scenarios:
+// 1. start at KMS 1
```

Comment:
Instead of `KMS 1` and `KMS 2`, let's have the actual kms provider names in the comments